psi4/driver/procrouting/proc_table.py

```diff
@@ -246,6 +246,8 @@
         procedures['energy']['sapt0-' + disp] = proc.run_sapt
         procedures['energy']['fisapt0-' + disp] = proc.run_fisapt
 
+procedures['energy']['sapt0-d4'] = proc.run_sapt
```

Comment:
Could this be leftover and unneeded? I'd think the block above could catch and define this.

Should fisapt0-d4 work?

Reply:
I have added a test for fsapt-d4 under `./tests/fsapt-d4` following similar tests of `./tests/fsapt-terms` and `/tests/fsaptd-terms`.